scheduler/scheduler.go

```diff
@@ -322,7 +343,10 @@ func (scheduler *Scheduler) Schedule(schedTime uint) {
 }
 
 func (scheduler *Scheduler) startClone(ff *FlowFunction) bool {
-	index := scheduler.getCoreIndex(false)
+	index, err := scheduler.getCoreIndex(false)
+	if err != nil {
+		common.LogError(common.Debug, err)
+	}
 	if index != -1 {
```

Comment:
I think this check is not necessary now. If err != nil we exit earlier on 348 line, otherwise index not -1

Reply:
Also 'else' case below cannot ever execute

Reply:
I changed getCoreIndex, it can return -1, nil (if so, error is not fatal, else log error and exit)